int64 blue_dis
int64 blue_ang1
int64 blue_ang2

int64 yellow_dis
int64 yellow_ang1
int64 yellow_ang2
